README.md

```diff
@@ -1,2 +1,5 @@
 # java-sandbox
 Java Sandbox for R&amp;D
+
+# Requirements:
+- Java JRE&sol;JDK must be installed and configured on your PC
```

Comment:
please update requirements according to Spring Boot official documentation, hence not all JDK version are allowed
<img width="589" alt="image" src="https://user-images.githubusercontent.com/6797490/226177545-f0f7c10e-d1e5-451d-a7b0-c982d43991e1.png">

Reply:
done